### ISSUE TITLE
5.x docker doesn't report downloadId correctly

### ISSUE THREAD

**Issue by cangencer** [hazelcast/hazelcast-docker#290]
looking at phone homes - I only see Maven or Distribution.

**Comment by leszko** [comment]
In 5.0 I see we use Hazelcast CLI to start the Hazelcast instance. @alparslanavci do you know how to change the pardotId while running Hazelcast using CLI?

I see now there are two `hazelcast-download.properties`:
```
$ find ./ | grep hazelcast-download.properties
./hazelcast-download.properties
./lib/hazelcast-download.properties
```

@alparslanavci any ideas?

**Comment by alparslanavci** [comment]
@leszko normally, the `hazelcast-download.properties` is located under `bin/download/` folder for CLI installations.

**Comment by leszko** [comment]
I checked the issue and actually Docker image does not use Hazelcast CLI. I was wrong here. 

I found the issue. So, the version `BETA-1` was manually built by Hasan, so he might have forgotten to set the `HZ_VERSION` arg while building the Docker image. @hasancelik could you check?

I also see that the snapshot version does not have the pardotId configured correctly. It may be that `HZ_VERSION` is not also set up correctly. @hasancelik could you also check that?

**Comment by cangencer** [comment]
I just tried with latest image, and still get `maven` (cc @frant-hartm )

**Comment by cangencer** [comment]
Also tried 5.1-SNAPSHOT (latest-snapshot tag) - that correctly reported as `docker`

**Comment by frant-hartm** [comment]
This will be fixed in the upcoming 5.0.1 image.
